Component/Mass/src/sofa/component/mass/MeshMatrixMass.h

```diff
@@ -230,10 +234,14 @@ class MeshMatrixMass : public core::behavior::Mass<DataTypes>
         {
             auto splitTemplates = sofa::helper::split(std::string(arg->getAttribute("template")), ',');
             if (splitTemplates.size() > 1)
-            {
-                msg_warning() << "MassType is not required anymore and the template is deprecated, please delete it from your scene." << msgendl
-                    << "As your mass is templated on " << DataTypes::Name() << ", MassType has been defined as " << sofa::helper::NameDecoder::getTypeName<MassType>() << " .";
-                msg_warning() << "If you want to set the template, you must write now \"template='" << DataTypes::Name() << "'\" .";
+            {   
+                // check if the given 2nd template is the deprecated MassType one
+                if (splitTemplates[1] == "float" || splitTemplates[1] == "double" || splitTemplates[1].find("RigidMass") != std::string::npos)
```

Comment:
you're lucky that the template you removed, and the one you added are incompatible 😝

Reply:
:whistle: 🙃